### ISSUE TITLE
Removing DataMaskingAttribute doesn't generate any change in the migration

### ISSUE THREAD

**Issue by clement911** [nikitasavinov/EntityFrameworkCore.Extensions#60]
Steps to repro:
1) Add `[DataMasking(MaskingFunction = "partial(3, \"xxxxxx\", 0)")]` to property
2) Generate migration and observe generated AlterColumn in migration
3) Remove attribute
4) Generate migration and observe the migration is empty.

**Comment by clement911** [comment]
Using version 3.1.2

**Comment by nikitasavinov** [comment]
Hi @clement911 , thank you for your feedback! It's nice (and a bit of surprise) to see this library used.

I reproduced the issue, and unfortunately there is indeed a bug with attributes. The workaround for now is to use the fluent API:
```  csharp
modelBuilder.Entity<Customer>()
.Property(t => t.Phone)
.HasAnnotation(AnnotationConstants.DynamicDataMasking, MaskingFunctions.Partial(2, "XX-XX", 1));```

I will tackle this issue with attributes, but only somewhere around the weekend.

**Comment by clement911** [comment]
No problem, I can definitely wait until next week.
By the way, have you considered creating a PR to merge the data masking migration code into EF Core?
I believe this could be useful to a lot of people.

**Comment by nikitasavinov** [comment]
Hey @clement911 , unfortunately fixing this issue will take more time than I originally thought. 
So I released a new package (3.2.2) which removes DataMaskingAttribute (breaking change). 

To add data masking you can use `HasDataMask` function (it works well with migrations)
``` csharp
    protected override void OnModelCreating(ModelBuilder modelBuilder)
    {
        base.OnModelCreating(modelBuilder);
        
        modelBuilder.Entity<Customer>().Property(t => t.Surname).HasDataMask(MaskingFunctions.Default());
```

>By the way, have you considered creating a PR to merge the data masking migration code into EF Core?

Thanks! Maybe one day 😃
